addons/web_tour/static/src/tour_service/tour_pointer_state.js

```diff
@@ -144,7 +144,16 @@ export function createPointerState() {
                         });
                         return;
                     }
-                    const { x, y, width, height } = scrollParent.getBoundingClientRect();
+                    let { x, y, width, height } = scrollParent.getBoundingClientRect();
+
+                    // If the scrolling element is within an iframe the offsets
+                    // must be computed taking into account the iframe.
+                    const iframeEl = scrollParent.ownerDocument.defaultView.frameElement;
+                    if (iframeEl) {
+                        const iframeOffset = iframeEl.getBoundingClientRect();
+                        x += iframeOffset.x;
+                        y += iframeOffset.y;
+                    }
```

Comment:
Was this problem only introduced in master? :thinking: Should not this be in previous version?

Reply:
> Was this problem only introduced in master? 🤔 Should not this be in previous version?

:+1: Indeed, I create a new PR